docs/source/changelog.rst

```diff
@@ -8,6 +8,22 @@ All notable changes to this project will be documented in this file.
 `Unreleased`_
 -------------
 
+`0.21.0`_ -- (unreleased)
+-------------------------
+
+Added
+^^^^^
+* Error |NoStateBackend| raised when is tried to access a |StateBackend| in a broker without it
+* Method ``chunk`` in ``commons.py`` return an iterable over a list of length ``size``
```

Comment:
As you can see in setup.py, remoulade.common is not included in the package. that mean all functions inside it are private to remoulade. So no need to add it to the changelog.